documentation/modules/managing/con-consumer-config-properties.adoc

```diff
@@ -6,11 +6,18 @@
 = Kafka consumer configuration tuning
 
 [role="_abstract"]
-Use a basic consumer configuration with optional properties that are tailored to specific use cases.
-
+Use configuration properties to optimize the performance of Kafka consumers.
+You can use standard Kafka consumer configuration options.
 When tuning your consumers your primary concern will be ensuring that they cope efficiently with the amount of data ingested.
 As with the producer tuning, be prepared to make incremental changes until the consumers operate as expected.
 
+When tuning a consumer, consider the following aspects carefully, as they significantly impact its performance and behavior:
+
+Scaling:: Consumer groups enable parallel processing of messages by distributing the load across multiple consumers. Properly configuring consumer groups can enhance scalability and throughput.
```

Comment:
I think this needs to mention partitions. Without them, this is confusing as users might not understand how the consumer groups scale.

Reply:
Agree with @scholzj. We should say that the number of topic partitions determines the max level of parallelism that you can achieve, because one partition can only be assigned to one consumer in a consumer group. That said, a single consumer can offload message processing to a thread pool when ordering is not mandatory, but this is not supported by the Kafka library.